components/arm/arm.go

```diff
@@ -365,34 +364,13 @@ func Move(ctx context.Context, r robot.Robot, a Arm, dst spatialmath.Pose) error
 
 // Plan is a helper function to be called by arm implementations to abstract away the default procedure for using the
 // motion planning library with arms.
-func Plan(
-	ctx context.Context,
-	r robot.Robot,
-	a Arm,
-	dst spatialmath.Pose,
-) ([][]referenceframe.Input, error) {
-	// build the framesystem
-	fs, err := framesystem.RobotFrameSystem(ctx, r, nil)
-	if err != nil {
-		return nil, err
-	}
-	armName := a.ModelFrame().Name()
-	destination := referenceframe.NewPoseInFrame(armName+"_origin", dst)
-
-	// PlanRobotMotion needs a frame system which contains the frame being solved for
-	if fs.Frame(armName) == nil {
-		armFrame := a.ModelFrame()
-		jp, err := a.JointPositions(ctx, nil)
-		if err != nil {
-			return nil, err
-		}
-		return motionplan.PlanFrameMotion(ctx, r.Logger(), dst, armFrame, armFrame.InputFromProtobuf(jp), defaultArmPlannerOptions, nil)
-	}
-	solutionMap, err := motionplan.PlanRobotMotion(ctx, destination, a.ModelFrame(), r, fs, nil, defaultArmPlannerOptions, nil)
+func Plan(ctx context.Context, logger golog.Logger, a Arm, dst spatialmath.Pose) ([][]referenceframe.Input, error) {
```

Comment:
[nit] it seems that `Plan` is only used by `Move`
why not just get rid of the `Plan` function?

Reply:
It's a public, externally useable function that returns the plan without executing it. This is useful to be able to do.

Reply:
makes sense - resolving this convo.